Eevee.Sleep.Bot/Modules/SlashCommands/ChangeRoleSlashModule.cs

```diff
@@ -0,0 +1,58 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for role things.")]
+public class ChangeRoleSlashModule : InteractionModuleBase<SocketInteractionContext> {
```

Comment:
Add a command `role track` to add the role to track the ownership.
This command will create/link all role records to an user when ran.

For example, user A and B have role RA. When `role track @RA` is ran, role record of user A and B should be updated to record that they have role `@RA`.

Reply:
Is this easier to operate if I prepare this in a command?
I was wondering if it's unnecessary since I can add roles from the UI of discord

Is this the same as `[SlashCommand("save", "Save all users roles")]` I assumed?

Reply:
Adding roles on Discord isn't the same as asking the bot to track the role addition/deletion tho, but yeah it has the same idea of what I commented on the `save` command.

Reply:
made role track command
[e7e432f](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/e7e432fec885e372b507c4eeed28c0373f387639)